nipoppy/workflows/dataset_init.py

```diff
@@ -146,49 +146,54 @@ def _init_manifest_from_bids_dataset(self) -> None:
 
         self.logger.info("Creating a manifest file from the BIDS dataset content.")
 
-        for ppt in participant_ids:
+        for bids_participant_id in bids_participant_ids:
 
-            session_ids = sorted(
+            bids_session_ids = sorted(
                 [
                     x.name
-                    for x in (self.layout.dpath_bids / ppt).iterdir()
+                    for x in (self.layout.dpath_bids / bids_participant_id).iterdir()
                     if x.is_dir() and x.name.startswith(BIDS_SESSION_PREFIX)
                 ]
             )
-            if len(session_ids) == 0:
+            if len(bids_session_ids) == 0:
                 # if there are no session folders
                 # we will add a fake session for this participant
-                session_ids = [FAKE_SESSION]
+                self.logger.warning(
+                    "Could not find session-level folder(s) for participant "
+                    f"{bids_participant_id}, using {FAKE_SESSION_ID} in the manifest"
+                )
+                bids_session_ids = [f"{BIDS_SESSION_PREFIX}{FAKE_SESSION_ID}"]
 
-            for ses in session_ids:
-                if ses == FAKE_SESSION:
+            for bids_session_id in bids_session_ids:
+                if bids_session_id == f"{BIDS_SESSION_PREFIX}{FAKE_SESSION_ID}":
                     # if the session is fake, we don't expect BIDS data
                     # to have session dir in the path
                     datatypes = sorted(
                         [
                             x.name
-                            for x in (self.layout.dpath_bids / ppt).iterdir()
+                            for x in (
+                                self.layout.dpath_bids / bids_participant_id
+                            ).iterdir()
                             if x.is_dir()
                         ]
                     )
                 else:
                     datatypes = sorted(
                         [
                             x.name
-                            for x in (self.layout.dpath_bids / ppt / ses).iterdir()
+                            for x in (
+                                self.layout.dpath_bids
+                                / bids_participant_id
+                                / bids_session_id
+                            ).iterdir()
                             if x.is_dir()
                         ]
                     )
 
-                # if there are no datatypes, raise warning and skip
-                if len(datatypes) == 0:
-                    self.logger.warning(
-                        f"Participant {ppt}, session {ses} has no datatypes. Skipping."
-                    )
-                    continue
```

Comment:
I forgot what we said about this -- did we still want a warning if there is no datatype?